tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/bootstrap-packer.ps1

```diff
@@ -0,0 +1,35 @@
+# https://docs.microsoft.com/en-us/windows/win32/winrm/winrm-powershell-commandlets
```

Comment:
Not sure if you need this script at all. If you use a public or imported image as base, [WinRM is automatically configured on boot](https://github.com/GoogleCloudPlatform/compute-image-windows/blob/6b42ff04aec5501b225d436c836ed73a1b8dd94d/sysprep/instance_setup.ps1#L125)

Reply:
This script is to configure a HTTPS listener using a local self signed cert as the default NTLM protocol used by WinRM is deprecated and not secure. The listener config and cert is torn down during the clean up script as the final part of the provisioner stage.

Going slightly off topic, WinRM between domain joined instances use Kerberos for auth, so is pretty safe but for workgroup/non domain joined environments like this, it'd be safer (and relatively easy) to encrypt traffic using HTTPS

Reply:
Not sure what the difference is... `instance_setup.ps1` also creates an HTTPS listener with a self-signed cert?